examples/go/tagging/gotag-fn.yml

```diff
@@ -0,0 +1,16 @@
+version: 1.0
+provider:
+  name: openfaas
+  gateway: http://127.0.0.1:8080
+functions:
+  gotag-fn:
+    lang: golang-http
+    handler: ./gotag-fn
+    image: fgold/gotag-fn:0.1.0
```

Comment:
Needs to be replaced with vmware repo and image ID (please discuss with @lamw )

Reply:
I went ahead and replaced it with vmware repo. I'll talk to @lamw about getting the image uploaded once I made all the changes for this review and test out the code.